types/TileServerConfig.ts

```diff
@@ -11,12 +11,6 @@ export interface TileServerConfig<T> {
    */
   maxZoomLevel?: number;
 
-  /**
-   * @description The tile resolution in pixels, default is 512, but try 256 if you
-   * are unsure what your mapping front-end library uses
-   */
-  resolution?: number;
```

Comment:
this can be made deprecated, so we don't need a breaking change yet. That way we can communicate to users that there is a new property they should look at when they upgrade, instead of just abruptly dropping it